data/campaigns/Delfadors_Memoirs/scenarios/16_Dark_Sky_Over_Weldyn.cfg

```diff
@@ -104,7 +104,7 @@
         [/part]
         [part]
             background=story/portraits/young_delfador-large.png
-            story={CAPTION ( _ "Delfador")} + _ "There are three things we must do. First, we must organize our defenses as best we can. Second, we need to ask the elves to create a diversion, to distract and delay Iliah-Malal. The elves will not be able to fight the undead head on on the field, but they can harass them by striking from forests where the undead dare not follow. And last, I have been studying and I think I now know how to close Iliah-Malal’s portal to the Land of the Dead. I will need a small escort, but this must be done immediately."
+            story={CAPTION ( _ "Delfador")} + _ "There are three things we must do. First, we must organize our defenses as best we can. Second, we need to ask the elves to create a diversion that will distract and delay Iliah-Malal. The elves will not be able to fight the undead head-on on the field, but they can harass them by striking from forests where the undead dare not follow. And last, I have been studying the Book of Crelanu and I think I now know how to close Iliah-Malal’s portal to the Land of the Dead. I will need a small escort, but this must be done immediately."
```

Comment:
> The elves will not be able to fight the undead head-on on the field...

I know it was written this way in the original text before this PR, but re-reviewing this while updating localisation I think this might be better phrased as 'head-on **in** the field'. I recognise there's a subtle difference in meaning there, but at least it avoids the clumsy-sounding repetition of 'on'.

Any opinions on this?

Reply:
I think either way is fine; the point about avoiding repetition does have me leaning towards "in" though

Reply:
I don't really think there's a need to avoid repetition when grammar requires it (a common case is two consecutive "that"s), and this seems like it's such a case. Unless "in" better fits the intended meaning, I'd leave it as "on".

Reply:
I'm aware of those cases, but those are quite rare. Even disregarding the repetition aspect, I think it reads better to me with 'in', but perhaps I'm in the minority here. If people are comfortable to leave it as-is, so be it.

Reply:
https://owl.purdue.edu/owl/general_writing/grammar/prepositions/prepositions_of_location_at_in_on.html
That article explains it well.
One of the example sentences is "Three players are practicing on the field"

Reply:
I was thinking of a different context of 'field', not like playing on a football field but fighting out in the field.

But no matter, I've already said I'm in the minority here.

Reply:
https://glosbe.com/en/en/to%20meet%20on%20the%20battlefield%20%28idiom%29
Here's an site that has cited usage of fighting or meeting out _on_ a battlefield.
I suppose the field in the context of the dialog is indeed a battlefield.

Reply:
I'm not disputing the usage of 'on'. I am simply saying that in the context I am thinking of 'in' is more appropriate - which clearly isn't how others are reading it. Like field-testing a new weapon or whatever.

I can't see why this is generating so much discussion when the other suggestion is arguably more significant. Makes me wish I never brought it up, should have just changed it without asking.